org.qownnotes.QOwnNotes.json

```diff
@@ -0,0 +1,41 @@
+{
+	"app-id": "org.qownnotes.QOwnNotes",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.10",
+	"sdk": "org.kde.Sdk",
+	"command": "QOwnNotes",
+	"rename-desktop-file": "PBE.QOwnNotes.desktop",
+	"rename-icon": "QOwnNotes",
+	"finish-args": [
+		"--socket=x11",
+		"--share=network",
+		"--device=dri",
+		"--filesystem=home"
+	],
+	"modules": [
+		{
+			"name": "QOwnNotes",
+			"buildsystem": "simple",
```

Comment:
You can use buildsystem: qmake and use a source of type shell to modify release.h

Reply:
I think you can leave the subdir as is. Ideally, the make install step should install the appdata as well, but in the meantime, you can add it as another shell command source